modules/unsupported/mbstyle/src/test/resources/org/geotools/mbstyle/expressionParseTest.json

```diff
@@ -155,6 +155,54 @@
             "paint": {
               "text-color": ["to-rgba", "#006fde"]
             }
+        },
+        {
+            "id": "getExpression",
+            "type": "symbol",
+            "source": "composite",
+            "source-layer": "waterway",
+            "layout": {
+                "text-field": ["get", "name"]
+            },
+            "paint": {
+                "text-color": ["get", "color", {"color": "#006fde"}]
+            }
+        },
+        {
+            "id": "hasExpression",
+            "type": "symbol",
+            "source": "composite",
+            "source-layer": "waterway",
+            "layout": {
+                "text-field": ["has", "name"]
+            },
+            "paint": {
+                "text-color": ["has", "color", {"color":"#006fde", "text":"text"}]
+            }
+        },
+        {
+            "id": "atExpression",
+            "type": "symbol",
+            "source": "composite",
+            "source-layer": "waterway",
+            "layout": {
+                "text-field": "testrgba"
+            },
+            "paint": {
+                "text-color": ["at", 0, ["#006fde", "green", "blue"]]
+            }
+        },
+        {
+            "id": "lengthExpression",
+            "type": "symbol",
+            "source": "composite",
+            "source-layer": "waterway",
+            "layout": {
+                "text-field": ["length", [1, 2, 3, 4, 5, 6, 7, 8, 9]]
+            },
+            "paint": {
+                "text-color": ["length", "123456789"]
+            }
```

Comment:
Tests look good overall, but could do with more nested structures, e.g:

```
["at", 0, ["get", "text-color", {"text-color": ["length", "123456789"]}]]
```
(should return `"length"`, in theory)